### DISCUSSION TITLE
How to remove html attributes on production build

### DISCUSSION THREAD

**Discussion by kamontat** [web-infra-dev/rspack#7005] [Q&A]
### What problem does this feature solve?

Normally we would add `data-testid` for easier integration test, however I don't want this on production build to avoid web scraping.

### What does the proposed API of configuration look like?

Maybe on `builtins` config?

**Comment by hardfist** [comment]
it seems you can custom html process logic yourself

**Comment by kamontat** [comment]
@hardfist Do you have any example code?
